.github/workflows/status-checks.yml

```diff
@@ -5,33 +5,33 @@ name: Status checks
 
```

Comment:
Are the status-checks still needed with the changes to the other workflow?

Reply:
I've removed the build job :)